includes/Shopping/Shopify_Query.php

```diff
@@ -198,9 +199,12 @@ protected function get_products_query( string $search_term ): string {
 	 * @since 1.21.0
 	 *
 	 * @param string $search_term Search term to filter products by.
+	 * @param string $orderby sort field for query.
+	 * @param string $order ASC or DESC.
 	 * @return array|WP_Error Response data or error object on failure.
 	 */
-	protected function fetch_remote_products( string $search_term ) {
+	protected function fetch_remote_products( string $search_term, string $orderby = '', string $order = '' ) {
```

Comment:
```suggestion
	protected function fetch_remote_products( string $search_term, string $orderby, string $order ) {
```

No need for default values here since `get_search()` below always passes something